seedDB_hg38/cgds.sql

```diff
@@ -0,0 +1,849 @@
+--
```

Comment:
is there a difference in this cgds sql compared to hg37?

Reply:
There is none, but I guess there is between versions. Thought it would be good to put the tested version with the seed data.